homeassistant/components/august/binary_sensor.py

```diff
@@ -195,7 +195,9 @@ class AugustDoorBinarySensor(AugustEntityMixin, BinarySensorEntity):
 
     _attr_device_class = BinarySensorDeviceClass.DOOR
 
-    def __init__(self, data, device, description: BinarySensorEntityDescription):
+    def __init__(
+        self, data, device, description: BinarySensorEntityDescription
+    ) -> None:
```

Comment:
I am aware that all changes could be improved, and there is still at least one argument untyped, but I think it still makes sense to add more partial type hints.